include/osmium/io/detail/debug_output_format.hpp

```diff
@@ -220,25 +221,37 @@ namespace osmium {
                 void write_meta(const osmium::OSMObject& object) {
                     output_int(object.id());
                     *m_out += '\n';
-                    if (m_options.add_metadata) {
+                    if (m_options.add_metadata.version()) {
                         write_fieldname("version");
                         *m_out += "  ";
                         output_int(object.version());
+                    }
+                    if (m_options.add_metadata.any()) {
                         if (object.visible()) {
```

Comment:
If `version` is off, this will probably look wrong with a leading space before `visible` or `deleted`. Have to check the ouput.

Reply:
You are right.

With version:
```
header
  multiple object versions: no
  bounding boxes: 
  options: 
    generator = osmium/1.7.1
    version = 0.6

=============================================

node 1
  version:   4 visible
  user:      2 "foo"
  lon/lat:   10,50

```

Without version:

```
header
  multiple object versions: no
  bounding boxes: 
  options: 
    generator = osmium/1.7.1
    version = 0.6

=============================================

node 1
 visible
  user:      2 "foo"
  lon/lat:   10,50
```

Shall I move visibility on its own line? I think that we don't have to preserve backward compatibility here because this format is intended to be read by a human and not processed by a program (like OPL).

Reply:
Yeah. Lets move it behind the first line:
```
    node 1 visible
      version:   4
      user:      2 "foo"
```